src/main/java/uk/gov/hmcts/reform/civil/model/CaseDataParent.java

```diff
@@ -722,6 +722,7 @@ public boolean isApplicantNotRepresented() {
     private DynamicList transferCourtLocationList;
     private NotSuitableSdoOptions notSuitableSdoOptions;
     private TocTransferCaseReason tocTransferCaseReason;
+    private String claimantBilingualLanguagePreference;
```

Comment:
Is it worth to change it to ENUM(ENGLISH and BOTH)?

Reply:
there is one related enum for Welsh, English , Welsh and English I did not wanted to add one more